### ISSUE TITLE
Reword "read more" link on home page

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1740]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Home page, Statement of Responsible Collection Description

### Description of error / warning

"Suspicious link text"
Link text contains extraneous text or may not make sense out of context.

Links, which are often read out of context, should clearly describe the destination or function of the link. Ambiguous text, text that does not make sense out of context, and extraneous text (such as "click here") can cause confusion and should be avoided.

The Algorithm... in English
A link (including alt text of linked images) contains the phrase "click here" or "click", or the link text is "click here", "here", "more", "more...", "details", "more details", "link", "this page", "continue", "continue reading", "read more", or "button".
Standards and Guidelines

### WCAG success criterion

[2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)

### WCAG conformance level

A

### Recommended fix

Reword the link text so that it is more descriptive of its destination when read out of context. Remove any extraneous text (such as "click here").

### Tool used to find the problem

wave

### Screenshot

<img width="530" height="435" alt="Image" src="https://github.com/user-attachments/assets/3d3d951f-9a06-4ac6-99e2-4f3f2bb6a1b4" />
